### PR TITLE
add new entries and translations

### PR DESCRIPTION
**Pull request by stardustichi**

### CODE REVIEW

File: src/data/skill-desc.csv [hunk 1]
```diff
@@ -31,6 +31,7 @@
 "最終コーナー以降で外から追い抜くと\n怪物的な力で、速度を上げる","<size=22>Increase speed with monstrous power from final corner onward when passing from outside</size>"
 "レース終盤に3回追い抜くと\n最終直線で速度がすごく上がる","<size=22>During the last leg of the race make three overtakes to strongly increase your speed on the final straight</size>"
 "最終コーナーで後ろから追い抜くと\n女帝たる走りで速度が上がる","<size=22>Increase speed when passing in the back in last corner</size>"
+"レース終盤のコーナーで先頭にいると\nこれ幸いと加速力を上げる","<size=22>On last corner, gain acceleration when leading in front</size>"
```
Comment (robflop) [inline src/data/skill-desc.csv:34]: "Leading in front" sounds a bit unclear imo (how far "front"? and such). 先頭 means the vanguard, in this case the very front (1st position). I'd incorporate that.

File: src/data/skill-desc.csv [hunk 2]
```diff
@@ -391,6 +392,7 @@
 "最終コーナー以降で外から追い抜くと\n速度をわずかに上げる","<size=22>Slightly Increase speed from final corner onward when passing from outside</size>"
 "レース終盤に3回追い抜くと\n最終直線で速度が少し上がる","<size=22>Overtake 3 times in Final Leg to lightly increase speed in the final straight</size>"
 "最終コーナーで後ろから追い抜くと\n速度がわずかに上がる","<size=22>Your speed increases slightly when passing from the back in the last corner</size>"
+"レース終盤のコーナーで先頭にいると\n加速力をわずかに上げる","<size=22>On last corner, slightly gain acceleration when leading in front</size>"
```
Comment (robflop) [inline src/data/skill-desc.csv:395]: Same as the previous comment about 先頭

File: src/data/trainer-title-requirements.csv [hunk 3]
```diff
@@ -195,7 +199,11 @@
 "ナリタブライアンのファン数を\n累計5000万人獲得しよう","Reach 50 million total fans for Narita Brian"
 "ナリタブライアンのファン数を\n累計1億人獲得しよう","Reach 100 million total fans for Narita Brian"
 "イベント期間限定ミッションを\nすべてクリアしよう","Clear all event limited missions"
-"タウラス杯 グレードリーグ\nAグループで優勝しよう","Win Group A in the Grade League of the Taurus Hai"
-"タウラス杯 グレードリーグ\nBグループで優勝しよう","Win Group B in the Grade League of the Taurus Hai"
-"タウラス杯 オープンリーグ\nAグループで優勝しよう","Win Group A in the Open League of the Taurus Hai"
-"タウラス杯 オープンリーグ\nBグループで優勝しよう","Win Group B in the Open League of the Taurus Hai"
+"タウラス杯で伝説的な成績を残そう","Leave a legendary achievement in the Taurus Hai"
```
Comment (robflop) [inline src/data/trainer-title-requirements.csv:202]: 残す is better translated as "achieve" here imo, as that sounds much more idiomatic and does not lose meaning anyway. If someone achieved something, they left the achievement behind for others to see, so it's exchangeable in that direction. 成績 you translated as achievement, but result or record would fit better here (based on literal meaning).

So I would go for "Achieve legendary results in the Taurus Hai" here, and similarly adjust all the others below this line too.

File: src/data/uma-nickname-requirements.csv [hunk 4]
```diff
@@ -140,3 +140,4 @@
 "全レースで2着以内で、\n菊花賞、天皇賞（春）、宝塚記念を勝利し、\n目標を全て達成し育成を完了する","<size=22>Finish ≥2nd in every single race\nWin the Kikka Sho, Tenno Sho (Spring), and Takarazuka Kinen\nClear all objectives and complete the training</size>"
 "作戦「追込」で皐月賞でラストの直線で4人以上を抜いて勝利し、\n作戦「追込」で天皇賞（春）を勝利し、\n基礎能力[根性]が1200以上になる。","<size=22>Win the Satsuki Sho as a chaser by breaking through ≥4 people in the last straight\nWin the Tenno Sho (Spring) as a chaser\nReach ≥1200 guts</size>"
 "皐月賞を3 1/2バ身差以上で勝利し、\n日本ダービーを5バ身差以上で勝利し、\n菊花賞を7バ身差以上で勝利し、有馬記念を二連覇する","<size=22>Win the Satsuki Sho, Japan Derby, and Kikka Sho with\ndifference of ≥3 1/2 lengths, ≥5 lengths, and ≥7 lengths\nand win the Arima Kinen twice</size>"
+"皐月賞、菊花賞、有馬記念（クラシック級）、天皇賞（春）を\n作戦「逃げ」で勝利し、うち菊花賞はスタート200m地点から\nゴールまで1番手のままで勝利する。","<size=22>As a runner, win the Satsuki Sho, Kikka Sho, Arima Kinen (2nd year), and Tenno Sho (Spring).\nIn the Kikka Sho, need to stay in the 1st position 200m from start until finish line</size>"
```
Comment (robflop) [inline src/data/uma-nickname-requirements.csv:143]: The "need to" at the end can be removed here to shorten it a bit and still stay accurate.

File: src/data/uma-title.csv [hunk 5]
```diff
@@ -18,6 +18,7 @@
 "[ロード・オブ・エンペラー]","[Road of Emperor]"
 "[エンプレスロード]","[Empress Road]"
 "[クエルクス・キウィーリス]","[Quercus Civilis]"
+"[あおぐもサミング]","[Blue Sky Thumbing]"
```
Comment (robflop) [inline src/data/uma-title.csv:21]: あおぐも (青雲, which can also be read as せいうん, hah) has a secondary meaning here as "high rank" or "erudition", but i'd keep it translated as "Blue Sky". The double meaning can't be made justice in english properly.

(To clarify, this is just a comment, not a request to change anything)

**Review by robflop — CHANGES_REQUESTED** [review]

File: src/data/trainer-title-requirements.csv (hunk 3 above)
Comment (stardustichi) [reply src/data/trainer-title-requirements.csv:202]: makes sense, i was thinking the same but i doubted myself

File: src/data/skill-desc.csv (hunk 1 above)
Comment (stardustichi) [reply src/data/skill-desc.csv:34]: thanks, i had a feeling something is missing, just couldnt find the word!

Comment (robflop) [reply src/data/skill-desc.csv]: I'd personally change this to simply be "when in 1st place", words like "front" and "very front" still feel a bit vague.

Specifically, something like "When in 1st place in the last corner, gain acceleration".

Comment (stardustichi) [reply src/data/skill-desc.csv]: it should be good now right??

Comment (FabulousCupcake) [reply src/data/skill-desc.csv]: I'm late, but I think I started these terms; to me:
-  `leading` means being in the very front (`order<=1`)
- `in the front` means being in the front _side_ (`order_rate<=50`) or 
  - Same with `behind`
- `in the lead` means something like `order<=3` (i.e. top 3 lead)

We should probably document this somewhere for consistency and re-review all the existing translations.

Comment (robflop) [reply src/data/skill-desc.csv]: Good idea, i would follow this convention then and see about documenting it somewhere (via separate PR).

**Comment by FabulousCupcake** [comment]
@robflop changes has been made; could you approve if there's anything else that needs to be updated?

File: src/data/uma-nickname-requirements.csv [hunk 6]
```diff
@@ -140,3 +140,4 @@
 "全レースで2着以内で、\n菊花賞、天皇賞（春）、宝塚記念を勝利し、\n目標を全て達成し育成を完了する","<size=22>Finish ≥2nd in every single race\nWin the Kikka Sho, Tenno Sho (Spring), and Takarazuka Kinen\nClear all objectives and complete the training</size>"
 "作戦「追込」で皐月賞でラストの直線で4人以上を抜いて勝利し、\n作戦「追込」で天皇賞（春）を勝利し、\n基礎能力[根性]が1200以上になる。","<size=22>Win the Satsuki Sho as a chaser by breaking through ≥4 people in the last straight\nWin the Tenno Sho (Spring) as a chaser\nReach ≥1200 guts</size>"
 "皐月賞を3 1/2バ身差以上で勝利し、\n日本ダービーを5バ身差以上で勝利し、\n菊花賞を7バ身差以上で勝利し、有馬記念を二連覇する","<size=22>Win the Satsuki Sho, Japan Derby, and Kikka Sho with\ndifference of ≥3 1/2 lengths, ≥5 lengths, and ≥7 lengths\nand win the Arima Kinen twice</size>"
+"皐月賞、菊花賞、有馬記念（クラシック級）、天皇賞（春）を\n作戦「逃げ」で勝利し、うち菊花賞はスタート200m地点から\nゴールまで1番手のままで勝利する。","<size=22>As a runner, win the Satsuki Sho, Kikka Sho, Arima Kinen (2nd year), and Tenno Sho (Spring).\nIn the Kikka Sho, stay in the 1st position 200m from start until finish line</size>"
```
Comment (robflop) [inline src/data/uma-nickname-requirements.csv:143]: This is a bit off. 「200m地点から」means "from the 200m line on", so it should be "from the 200m line until the goal", or something like it.

**Review by robflop — CHANGES_REQUESTED** [review]

File: src/data/uma-nickname-requirements.csv (hunk 6 above)
Comment (stardustichi) [reply src/data/uma-nickname-requirements.csv]: that means i need to change the other one here somewhere as well, but lets save it for next time, when we do the translations re-review

**Review by robflop — APPROVED** [review]

**Comment by FabulousCupcake** [comment]
🙌

**Pull request merged**